docs/design/protobuf-grpc-service.md

```diff
@@ -44,9 +44,11 @@ In order to better define resources at the API level, a few proto files will be
 - Some of the Kubernetes API like `tolerance` and `node affinity` are too complicated to be converted to an API.
 - We want to leave some flexibility to use database to store history data in the near future (for example, pagination, list options etc).
 
-We end up propsing a simple and easy API which can cover most of the daily requirements.
+To resolve these issues, we provide a simple API which can cover most common use-cases. 
 
-```
+For example, the protobuf definition of the `RayCluster`:
```

Comment:
I do not have any experience with grpc and protobuf. What's the relationship between these two and APIServer?

Is it a typical pattern to use a lot of code (e.g. `message ListClustersResponse { ... }`) for the documents related to grpc or protobuf?

Thank you!

Reply:
Here `.proto` in directory `/kuberay/proto` is the definition of the grpc interface of the apiserver, while apiserver provide http and rpc calling for the RESTful APIs. I was just following the original pattern and update the fields to the latest version, I have no preference on whether we need to add this example in the documents since we have generated swagger files for detail apis definition.

Reply:
Got it. Thank you for the explanation!